bulk_processing/find_invalid_address_case_id_by_uprn.py

```diff
@@ -37,15 +40,10 @@ def get_case_id_from_case_api(uprn, line_number):
     return [case['id'] for case in result]
 
 
-def write_invalid_addresses_case_id_file(case_id_list, file_to_process):
-    address_delta_file = Path(f'invalid_addresses_{file_to_process.stem}.csv')
-
-    with open(address_delta_file, 'w') as out:
-        csv_out = csv.writer(out)
-        csv_out.writerow(['case_id', 'reason'])
-        for case in case_id_list:
-            csv_out.writerow([case, 'ADDRESS_DELTA'])
-    return address_delta_file
+def write_invalid_addresses_case_id_file(case_id, address_delta_file):
+    with open(address_delta_file, 'a') as out:
```

Comment:
This opens/closes the file for every line it writes, best to open the whole time it's being written